app/controllers/authentication_controller.rb

```diff
@@ -52,4 +57,8 @@ def auth0_sign_out_url
 
     URI::HTTPS.build(host: Settings.auth0.domain, path: "/v2/logout", query: request_params.to_query).to_s
   end
+
+  def mock_gds_sso_sign_out_url
```

Comment:
Yes, that's the name of the provider in gds-sso (see https://github.com/alphagov/gds-sso/blob/a9c2a018dbebd07c35682b1523a2ad27a41d1185/lib/gds-sso/warden_config.rb#L66-L89). It is meant to be used for both local development and testing, I think, although I agree the naming is a little confusing. It would be best if we didn't rely on gds-sso for local testing, but an obvious alternative has yet to present itself...